style.css

```diff
@@ -0,0 +1,51 @@
+@import url("https://fonts.googleapis.com/css2?family=Bakbak+One&family=Inter:wght@100..900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Bakbak One", "Inter";
+}
+.nav-bar {
+  position: fixed;
+  width: 100%;
+  height: 14vmin;
+  background-color: white;
+
+  display: flex;
+  justify-content: flex-start;
+  align-items: center; 
+  position: relative;
+  /* justify-content: space-between;  */
+  box-shadow: 1px 1px 10px black; 
+  z-index: 100;
+}
+.nav-icon img {
+  height: 7vmin;
```

Comment:
Avoid adjusting the both the dimensions of the icons as it destroys the aspect ratio stick to only one either adjust the height or adjust the width